doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -240,12 +298,17 @@ References and Footnotes
 .. _Open Publication License: https://www.opencontent.org/openpub/
 
 .. _`xsimd`: https://xsimd.readthedocs.io/en/latest/
+.. _`Pixman`: https://gitlab.freedesktop.org/pixman
+.. _`VOLK`: https://www.libvolk.org/doxygen/index.html
```

Comment:
Lib `VOLK` is more like [Intel IPP,](https://software.intel.com/en-us/ipp) doesn't provide low access to any intrinsics, only ready use kernels and this kind of design proved that isn't flexible at all.

Reply:
I think I related to this on line 239 above. Does it need more clarification?

Reply:
No, it's fine, but I don't understand the idea behind mention it, Is that a purpose to use `libvolk` or `intel IPP` as embedded kernels for ufuncs.?

Reply:
No. This is a random collection of multi-cpu techniques, for reference. None of these are being considered for implementation.